backend/app/tests/conftest.py

```
@@ -1,17 +1,38 @@
+from collections.abc import Generator
+
 import pytest
 from fastapi.testclient import TestClient
-from sqlmodel import Session
+from sqlmodel import Session, create_engine
```

Ruff (0.12.2)

`sqlmodel.create_engine` imported but unused

Remove unused import: `sqlmodel.create_engine`

(F401)